x/liquidity/spec/04_messages.md

```diff
@@ -1,95 +1,107 @@
-	PoolTypeId          uint32         // id of the liquidity pool type of this new liquidity pool
-	DepositCoins 	    sdk.Coins      // deposit coins for initial pool deposit into this new liquidity pool
+    PoolCreatorAddress  string         // account address of the origin of this message
+    PoolTypeId          uint32         // id of the new liquidity pool
+    DepositCoins         sdk.Coins      // deposit initial coins for new liquidity pool
 }
 ```
 
-**Validity check**
+### Validity checks
+
+Validity checks are performed for MsgCreatePool messages. The transaction that is triggered with `MsgCreatePool` fails if:
 
-- `MsgCreatePool` fails if
-  - `PoolCreator` address does not exist
-  - `PoolTypeId` does not exist in parameters
-  - there exists duplicated `LiquidityPool` with same `PoolTypeId` and Reserve Coin Denoms
-  - if one or more coins in ReserveCoinDenoms do not exist in `bank` module
-  - if the balance of `PoolCreator` does not have enough amount of coins for `DepositCoins`
-  - if the balance of `PoolCreator` does not have enough amount of coins for paying `PoolCreationFee`
+- `PoolCreator` address does not exist
+- `PoolTypeId` does not exist in parameters
+- A duplicate `LiquidityPool` with same `PoolTypeId` and `ReserveCoinDenoms` exists
+- One or more coins in `ReserveCoinDenoms` do not exist in `bank` module
+- The balance of `PoolCreator` does not have enough amount of coins for `DepositCoins`
+- The balance of `PoolCreator` does not have enough coins for `PoolCreationFee`
 
 ## MsgDepositWithinBatch
 
-Deposit coins to the specified liquidity pool.
+Coins are deposited in a batch to a liquidity pool with the `MsgDepositWithinBatch` message.
 
 ```go
 type MsgDepositWithinBatch struct {
-	DepositorAddress    string         // account address of the origin of this message
-	PoolId              uint64         // id of the liquidity pool where this message is belong to
-	DepositCoins 	    sdk.Coins      // deposit coins of this pool deposit message
+    DepositorAddress    string         // account address of depositor that originated this message
+    PoolId              uint64         // id of the liquidity pool to receive deposit
+    DepositCoins         sdk.Coins      // deposit coins
 }
 ```
 
-**Validity check**
+## Validity checks
 
-- `MsgDepositWithinBatch` failes if
-  - `Depositor` address does not exist
-  - `PoolId` does not exist
-  - if the denoms of `DepositCoins` are not composed of `ReserveCoinDenoms` of the `LiquidityPool` with given `PoolId`
-  - if the balance of `Depositor` does not have enough amount of coins for `DepositCoins`
+The MsgDepositWithinBatch message performs validity checks. The transaction that is triggered with the `MsgDepositWithinBatch` message fails if:
+
+- `Depositor` address does not exist
+- `PoolId` does not exist
+- The denoms of `DepositCoins` are not composed of existing `ReserveCoinDenoms` of the specified `LiquidityPool`
```

Comment:
is this validity check the same validity check on line 69 "- The denom of `PoolCoin` are not equal to the `PoolCoinDenom` of the `LiquidityPool`"
I think we're just saying "the denom does not exist in the liquidity pool" right?